packages/view-types/feature-list/src/FeatureListSubscriber.js

```diff
@@ -36,6 +36,7 @@ export function FeatureListSubscriber(props) {
     title: titleOverride,
     enableMultiSelect = false,
     showTable = false,
+    sort = 'alphabetical',
```

Comment:
Would you be able to add a JSDoc comment about which values are valid ('alphabetical' vs. all else), for our future selves/developers?